Gemfile

```diff
@@ -13,6 +13,8 @@ gem 'omniauth-github'
 gem 'bootstrap-sass'
 gem 'font-awesome-rails'
 gem 'kaminari'
+gem 'octicons'
+gem 'octicons_helper'
```

Comment:
You actually only need `octicons_helper` because `octicons_helper` installs `octicons` as a dependency

https://github.com/primer/octicons_helper/blob/7ba0d858139a3d1e472115049dff2fa8eeda7378/octicons_helper.gemspec#L16